hazelcast/src/test/java/com/hazelcast/instance/impl/HazelcastBootstrapTest.java

```diff
@@ -36,15 +38,30 @@
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class HazelcastBootstrapTest {
 
+    @AfterClass
+    public static void teardown() {
+        Hazelcast.bootstrappedInstance().shutdown();
+    }
+
     @Test
-    public void bootstrappedInstance() {
+    public void testHazelcast_bootstrappedInstance() {
         HazelcastInstance hz = Hazelcast.bootstrappedInstance();
         JetInstance jet = hz.getJetInstance();
+        executeWithBootstrappedInstance(jet);
+    }
+
+    @Test
```

Comment:
yes not related, there was a race when shutting down the instance between JetBootstrapTest and HazelcastBootstrapTest. I've merged them to a single test so that we shutdown the created instance once at the end of the test.